models.py

```diff
@@ -0,0 +1,30 @@
+from sqlalchemy import (
+    Column,
+    Integer,
+    String,
+    Date,
+    ForeignKey,
+)
+from sqlalchemy.orm import relationship
+
+from database import Base
+
+
+class DBAuthor(Base):
+    __tablename__ = "authors"
+
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(String(255), unique=True, nullable=False)
+    bio = Column(String(511), nullable=False)
+    books = relationship("DBBook", back_populates="author")
+
+
+class DBBook(Base):
```

Comment:
Rename it to Book. We never declare DBBook to show that is model. In fastapi we use model.Author, schemas.Author etc.